samples/performance/texture_compression_basisu/texture_compression_basisu_tutorial.md

```diff
@@ -0,0 +1,242 @@
+<!--
+- Copyright (c) 2021, Sascha Willems
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+-->
+
+# Using Basis Universal supercompressed GPU texture codec with Vulkan
+
+## Overview
+
+This tutorial, along with the accompanying example code, demonstrates how to use Basis universal supercompressed GPU textures in a Vulkan application.
+
+## The KTX2 format
+[KTX](https://www.khronos.org/ktx/) is a GPU texture **container format** for storing different texture types (2D, cubemap, etc.) and texture formats (uncompressed and compressed). Version 2.0 added support for Basis Universal supercompressed textures.
+
+## Basis Universal
+
+[Basis Universal](https://github.com/BinomialLLC/basis_universal) is a supercompressed GPU texture data interchange system that implements the UASTC and ETC1S compressed formats that server as **transmission formats**. Both can be quickly transcoded to a wide variety of GPU native compressed an uncompressed formats like RGB/RGBA, PVRTC1, BCn, ETC1, ETC2, etc. This means that unlike a KTX 2.0 file storing a BC3 textures, the data needs to be transcoded at runtime.
```

Comment:
UASTC and ETC1S compressed formats that **server** as **transmission formats**

Possible typo: server -> serve?

Reply:
compressed an uncompressed -> compressed and uncompressed

Reply:
Thanks for spellchecking. Fixed both typos :)